src/primitives/block.cpp

```diff
@@ -5,14 +5,17 @@
 
 #include "primitives/block.h"
 
+#include "streams.h"
 #include "hash.h"
```

Comment:
Let's sort them alphabetically

```suggestion
#include "hash.h"
#include "streams.h"
```